aiperf/gpu_telemetry/telemetry_manager.py

```
@@ -55,35 +57,38 @@ def __init__(
 
-        # [AIP-280] TODO: @ilana-n Get from user_config when telemetry config is added to CLI argument
-        self._dcgm_endpoints = [DEFAULT_DCGM_ENDPOINT]
+        # Normalize user_endpoints to always be a list
+        user_endpoints = user_config.server_metrics_url
+        if not user_endpoints:
+            user_endpoints = []
+        elif isinstance(user_endpoints, str):
+            user_endpoints = [user_endpoints]
+
+        if DEFAULT_DCGM_ENDPOINT not in user_endpoints:
+            self._dcgm_endpoints = [DEFAULT_DCGM_ENDPOINT] + user_endpoints
```

Ruff (0.13.1)

Consider `[DEFAULT_DCGM_ENDPOINT, *user_endpoints]` instead of concatenation

Replace with `[DEFAULT_DCGM_ENDPOINT, *user_endpoints]`

(RUF005)